api/provider/aws/service_read.go

```diff
@@ -65,24 +67,25 @@ func (s *ServiceProvider) Read(serviceID string) (*models.Service, error) {
 	return model, nil
 }
 
-func (s *ServiceProvider) readService(clusterName, serviceID string) (*ecs.Service, error) {
+func (s *ServiceProvider) readService(clusterName, fqServiceID string) (*ecs.Service, error) {
```

Comment:
I get the rationale for using `fqServiceID` here, but that is inconsistent with what we have already implemented - which is to use ecs/aws verbiage for helper functions that wrap aws api calls. In this case, the ecs verbiage is `serviceID`.

If we want to change that, it needs to be an all-or-nothing thing for consistencies sake.